src/components/AddNewProject.js

```diff
@@ -1,10 +1,28 @@
 import React from "react";
 import { useForm } from "react-hook-form";
+import axios from 'axios';
 
 const AddNewProject = () => {
     const { register, errors, handleSubmit } = useForm();
 
-    const onSubmit = data => console.log(data);
+    const onSubmit = (data) => {
+        // console.log(data);
+        let tech = data.language.split(',');
+
+        let project = {
+            "projectName": data.projectName,
+            "description": data.description,
+            "category": ["Frontend"],
+            "language": tech,
+            "userId": "1"
+        }
+
+        console.log(project);
+
+        axios.post('https://projecthunt-api.herokuapp.com/project', project)
```

Comment:
create a new file under `utils` directory, for e.g `api.js` and all the API calls can be moved there and you can export the function such as `createProject` which can take in data parameter and do API call over there and return a promise.